src/TwigComponent/src/ComponentRenderer.php

```diff
@@ -107,25 +107,27 @@ private function preRender(MountedComponent $mounted, array $context = []): PreR
     {
         $component = $mounted->getComponent();
         $metadata = $this->factory->metadataFor($mounted->getName());
+        $classProps = $metadata->isPublicPropsExposed() ? iterator_to_array($this->exposedVariables($component, $metadata->isPublicPropsExposed())) : [];
```

Comment:
This should be reverted: even when public props are not exposed, $this->exposedVariables() may return props (using attributes for instance)
```suggestion
        $classProps = iterator_to_array($this->exposedVariables($component, $metadata->isPublicPropsExposed()));
```